### PR TITLE
Validate vagrant command line parameters

### PR DESCRIPTION
**Pull request by ynixon**
Validate vagrant command line parameters and  force user to enter dev/stage/prod - to prevent mistakes and errors like #152

### CODE REVIEW

**Comment by lioramilbaum** [comment]
```
(venv) Lioras-MacBook-Pro:devopsloft liora$ vagrant destroy -f dev
You must use 'vagrant destroy dev/stage/prod....'
Run 'vagrant status' to view VM names.
```

File: README.md [hunk 1]
```diff
@@ -105,4 +103,4 @@ vagrant up stage
 * !!! Don't forget to destroy the instance when you done to avoid unnecessary charges
 ```
 vagrant destroy --force stage
```
Comment (lioramilbaum) [inline README.md]: see my comment. It is not working.

Comment (ynixon) [reply README.md]: added check also for the 3rd parameter

**Comment by lioramilbaum** [comment]
LGTM

**Pull request merged**